plugins/module_utils/network/sonic/facts/static_routes/static_routes.py

```diff
@@ -0,0 +1,172 @@
+            except ConnectionError as exc:
+                module.fail_json(msg=str(exc), code=exc.code)
+            for resp in response:
+                if 'openconfig-network-instance:static-routes' in resp[1]:
+                    static_routes_dict = resp[1].get('openconfig-network-instance:static-routes', {})
+                    static_routes_dict['vrf'] = vrf_name
+                    all_static_routes.append(static_routes_dict)
+        return all_static_routes
+
+    def update_static_routes(self, data):
+        static_vrf_list = []
+        for static_route in data:
+            static_vrf_dict = {}
+            static_route_list = static_route.get('static', [])
+            vrf_name = static_route.get('vrf', None)
+            static_list = []
+            for static in static_route_list:
+                static_dict = {}
+                prefix = static.get('prefix', None)
+                next_hops = static.get('next-hops', None)
+                next_hop_list = next_hops.get('next-hop', [])
+                next_hops_list = []
+                for next_hop in next_hop_list:
+                    next_hop_dict = {}
+                    index_dict = {}
+                    inf_ref = next_hop.get('interface-ref', {})
+                    inf_ref_cfg = inf_ref.get('config', {})
+                    interface = inf_ref_cfg.get('interface', None)
+                    config = next_hop.get('config', {})
+                    next_hop_attr = config.get('next-hop', None)
+                    metric = config.get('metric', None)
+                    network_instance = config.get('network-instance', None)
+                    blackhole = config.get('blackhole', None)
+                    track = config.get('track', None)
+                    tag = config.get('tag', None)
+                    if blackhole:
+                        index_dict['blackhole'] = blackhole
+                    if interface:
+                        index_dict['interface'] = interface
+                    if network_instance:
+                        index_dict['network_instance'] = network_instance
+                    if next_hop_attr:
+                        index_dict['next_hop'] = next_hop_attr
+                    if index_dict:
+                        next_hop_dict['index'] = index_dict
+                    if metric:
+                        next_hop_dict['metric'] = metric
+                    if track:
+                        next_hop_dict['track'] = track
+                    if tag:
+                        next_hop_dict['tag'] = tag
+                    if next_hop_dict and next_hop_dict not in next_hops_list:
+                        next_hops_list.append(next_hop_dict)
+                if prefix:
+                    static_dict['prefix'] = prefix
+                if next_hops_list:
+                    static_dict['next_hops'] = next_hops_list
+                if static_dict and static_dict not in static_list:
```

Comment:
Same question here: How could the static_dict already be in the list? (The same question applies for the static_vrf_dict presence in the static_vrf_list, below)